README.md

```diff
@@ -1,55 +1,169 @@
 Mentoring XBlock
 ----------------
 
-This XBlock allows to automate the workflow of real-life mentoring, within an edX course.
+[![Build Status](https://travis-ci.org/open-craft/xblock-mentoring.svg?branch=master)](https://travis-ci.org/open-craft/xblock-mentoring)
+
+This XBlock allows to automate the workflow of real-life mentoring,
+within an edX course.
 
 It supports:
 
-* **Free-form answers** (textarea) which can be shared accross different XBlock instances (for example, to remind a student of an answer he gave before). Editable or read-only.
-* **Self-assessment quizzes** (multiple choice), to display predetermined feedback to a student based on his choices in the self-assessment. Supports rating scales and arbitrary answers.
-* **Progression tracking**, allowing to check that the student has completed the previous steps before allowing to complete a given XBlock instance. Provides a link to the next step to the student.
-* **Tables**, which allow to present answers from the student to free-form answers in a concise way. Supports custom headers.
-* **Data export**, to allow course authors to download a CSV file containing the free-form answers entered by the students
+* **Free-form answers** (textarea) which can be shared accross
+  different XBlock instances (for example, to remind a student of an
+  answer he gave before). Editable or read-only.
+* **Self-assessment MCQs** (multiple choice), to display predetermined
+  feedback to a student based on his choices in the
+  self-assessment. Supports rating scales and arbitrary answers.
+* **Progression tracking**, allowing to check that the student has
```

Comment:
'allowing to check' is awkward phrasing. To keep in line with the other items, it should probably be 'to allow checking'